src/test/java/org/gridsuite/study/server/SecurityAnalysisTest.java

```diff
@@ -91,6 +89,9 @@ public class SecurityAnalysisTest {
     private static final String SECURITY_ANALYSIS_STATUS_JSON = "\"CONVERGED\"";
     private static final String CONTINGENCIES_JSON = "[{\"id\":\"l1\",\"elements\":[{\"id\":\"l1\",\"type\":\"BRANCH\"}]}]";
 
+    public static final String SECURITY_ANALYSIS_PARAMETERS_JSON = "{\"lowVoltageAbsoluteThreshold\":0.0,\"lowVoltageProportionalThreshold\":0.0,\"highVoltageAbsoluteThreshold\":0.0,\"highVoltageProportionalThreshold\":0.0,\"flowProportionalThreshold\":0.1}";
+    public static final String SECURITY_ANALYSIS_PARAMETERS_JSON2 = "{\"lowVoltageAbsoluteThreshold\":0.0,\"lowVoltageProportionalThreshold\":0.0,\"highVoltageAbsoluteThreshold\":0.0,\"highVoltageProportionalThreshold\":0.0,\"flowProportionalThreshold\":0.2}";
```

Comment:
could you rename into `SECURITY_ANALYSIS_UPDATED_PARAMETERS_JSON` ?

It would be better to have all values different from the default ones.

Reply:
Done